projects/spot/grasp.py

```diff
@@ -1,30 +1,36 @@
 from home_robot.utils.config import get_config
-from home_robot_hw.env.spot_goat_env import SpotGoatEnv
 from spot_wrapper.spot import Spot, SpotCamIds
-from home_robot.agent.goat_agent.goat_agent import GoatAgent
 import numpy as np
 import cv2
 import time
 from spot_rl.models import OwlVit
 from spot_wrapper.spot import image_response_to_cv2 as imcv2
 
 class GraspController():
-    def __init__(self, spot=None, objects=[['ball', 'lion']], confidence=0.05, show_img=False, top_grasp=False, hor_grasp=False):
+    def __init__(self, config=get_config("projects/spot/configs/config.yaml"), spot=None, objects=[['ball', 'lion']], confidence=0.05, show_img=False, top_grasp=False, hor_grasp=False):
         self.spot = spot
         self.labels = [f'an image of {y}' for x in objects for y in x]
         self.confidence = confidence
         self.show_img = show_img
         self.top_grasp = top_grasp
         self.hor_grasp = hor_grasp
-        self.stow = np.deg2rad([0, -170, 120, 0, 75, 0])
-        self.owl = OwlVit(self.labels, self.confidence, self.show_img)
+        self.detector = OwlVit(self.labels, self.confidence, self.show_img)
```

Comment:
how does owlvit compare to eg groundedsam?

Reply:
Replacing owlvit with detic/groundedsam